packages/util/src/fs.ts

```diff
@@ -35,3 +36,15 @@ export function readJSON(path: PathLike): { [key: string]: any } {
   logger.warn(`File "${path}" does not exist, return empty object.`);
   return {};
 }
+
+export function transpileDirectory(path: string): Promise<void> {
+  logger.info(`Transpiling files in the directory: ${path} started.`);
+  return execa('tsc', { cwd: path })
+    .then(() => {
+      logger.info(`Transpiling files in directory: ${path} finished.`);
+    })
+    .catch(err => {
+      logger.error(`Error: Failed to generate js files: ${err}`);
+      process.exit(1);
+    });
```

Comment:
execa is not declared as `async` funciton so when I used await it was complaining.

Reply:
Are you sure, that it wasn't complaining that `transpileDirectory` has to be async? Because you should be able to even do things like this: `await 3;`.

Reply:
Are you sure, that it wasn't complaining that `transpileDirectory` has to be async? Because you should be able to even do things like this: `await 3;`.

Reply:
yes you are right